### DISCUSSION TITLE
Is there any parameter to control the size of the merged files?

### DISCUSSION THREAD

**Discussion by joaohps** [aws/aws-sdk-pandas#1199] [Q&A]
https://github.com/awslabs/aws-data-wrangler/blob/47fd897fef9fbcf9384916427dcfc2b01c58640e/tutorials/013%20-%20Merging%20Datasets%20on%20S3.ipynb

Is there any parameter to control the size of the created merged files?

**Comment by jaidisido** [comment]
There is no parameter to control the size of merged files, however, there is a `max_rows_by_file` parameter to control the number of rows: https://aws-data-wrangler.readthedocs.io/en/stable/stubs/awswrangler.s3.to_parquet.html#awswrangler.s3.to_parquet
